src/topic/management/commands/load_topics_from_openalex.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand
+
+from topic.models import Topic
+from utils.openalex import OpenAlex
+
+
+class Command(BaseCommand):
```

Comment:
Since were are starting a new app with `topic`, it would be good to have some tests for this command?

Reply:
Yes, I was planning on adding some tests today!